### ISSUE TITLE
Switch remaining jUnit 4 tests to jUnit jupiter

### ISSUE THREAD

**Issue by OlgaMaciaszek** [spring-cloud/spring-cloud-netflix#4011]

**Comment by SivaTharun** [comment]
@OlgaMaciaszek can i work on this issue.

**Comment by OlgaMaciaszek** [comment]
@SivaTharun , thanks for reaching out. @maniotie is already working on this issue for now. If not all modules are converted, I will get back to you here.

**Comment by OlgaMaciaszek** [comment]
@maniotie is now working on reducing method and class visibility after the conversion to JUnitJupiter.

**Comment by OlgaMaciaszek** [comment]
Done.

**Comment by ThomasTopuz** [comment]
Is anybody working on this?

**Comment by OlgaMaciaszek** [comment]
I don't think anybody is at the moment. They are only a few test left, but possibly the ones that were proving tricky during the previous work done by contributors. Would you like to work on it @ThomasTopuz ?

**Comment by deii** [comment]
The remaining tests use the deprecated `ModifiedClassPathRunner` class. It would be possible to replace it with `ModifiedClassPathExtension`, but this class has a package-access level. How could we solve this? Should I create an issue and patch in the spring-cloud-commons project?

 https://github.com/spring-cloud/spring-cloud-commons/blob/main/spring-cloud-test-support/src/main/java/org/springframework/cloud/test/ModifiedClassPathExtension.java#L49

**Comment by OlgaMaciaszek** [comment]
Right, @deii. Have created: https://github.com/spring-cloud/spring-cloud-commons/pull/1358.

**Comment by OlgaMaciaszek** [comment]
It's merged now. @deii @ThomasTopuz Let me know if any of you would like to work on this.

**Comment by Pjaygit** [comment]
@OlgaMaciaszek is this available to work?

**Comment by Pjaygit** [comment]
Sure. Would like to try @OlgaMaciaszek

**Comment by OlgaMaciaszek** [comment]
Sounds good, @Pjaygit. Please go ahead.

**Comment by OlgaMaciaszek** [comment]
Please add your changes against the `4.1.x` branch.

**Comment by Pjaygit** [comment]
@OlgaMaciaszek i am seeing issue while building the project Non-resolvable parent POM for 4.2.1-SNAPSHOT version. Am I missing something?

**Comment by david-vana** [comment]
Looks like nobody is working on this currently. Could you please check my PR @OlgaMaciaszek ?

**Comment by OlgaMaciaszek** [comment]
Thanks, @david-vana - will review this week.

**Comment by david-vana** [comment]
@OlgaMaciaszek My PR is agains 4.2.x but as I realized - it should be earliest maintained branch 4.1.x, right? Should I create new PR agains 4.1.x?

**Comment by david-vana** [comment]
New PR #4425 against 4.1.x ready for review.

**Comment by Shaanworkspace** [comment]
I'd like to work on this, is it available?

**Comment by OlgaMaciaszek** [comment]
Hi, @Shaanworkspace, it is. Will assign it to you.
